'npm run build' fails

@aminalhazwani reports a problem with this repo:

```
$ npm run build

> @googlefonts/badge@0.8.0 build /Users/Amin/Sites/friends-badge
> mkdir build; browserify index.js -s gfBadge -o build/googlefontsbadge.js; ./node_modules/minify/bin/minify build/googlefontsbadge.js build/googlefontsbadge.js

mkdir: build: File exists
sh: ./node_modules/minify/bin/minify: No such file or directory

npm ERR! Darwin 16.1.0
npm ERR! argv "/Users/Amin/.nvm/versions/v6.6.0/bin/node" "/Users/Amin/.nvm/versions/v6.6.0/bin/npm" "run" "build"
npm ERR! node v6.6.0
npm ERR! npm  v3.10.3
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! @googlefonts/badge@0.8.0 build: `mkdir build; browserify index.js -s gfBadge -o build/googlefontsbadge.js; ./node_modules/minify/bin/minify build/googlefontsbadge.js build/googlefontsbadge.js`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the @googlefonts/badge@0.8.0 build script 'mkdir build; browserify index.js -s gfBadge -o build/googlefontsbadge.js; ./node_modules/minify/bin/minify build/googlefontsbadge.js build/googlefontsbadge.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the @googlefonts/badge package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     mkdir build; browserify index.js -s gfBadge -o build/googlefontsbadge.js; ./node_modules/minify/bin/minify build/googlefontsbadge.js build/googlefontsbadge.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs @googlefonts/badge
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls @googlefonts/badge
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/Amin/Sites/friends-badge/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -14,6 +14,7 @@
     "eslint": "3.7.1",
     "eslint-config-google": "0.6.0",
     "harp-minify": "0.4.0",
+    "mkdirp": "0.5.1",
     "qunitjs": "2.0.1"
   },
   "eslintConfig": {
@@ -27,6 +28,6 @@
     "pretest": "npm run build",
     "posttest": "npm run lint",
     "test": "",
-    "build": "mkdir build; browserify index.js -s gfBadge -o build/googlefontsbadge.js; ./node_modules/.bin/minify build/googlefontsbadge.js build/googlefontsbadge.js"
+    "build": "mkdirp build; browserify index.js -s gfBadge -o build/googlefontsbadge.js; ./node_modules/.bin/minify build/googlefontsbadge.js build/googlefontsbadge.js"
   }
 }

--- a/package.json
+++ b/package.json
@@ -27,6 +27,6 @@
     "pretest": "npm run build",
     "posttest": "npm run lint",
     "test": "",
-    "build": "mkdir build; browserify index.js -s gfBadge -o build/googlefontsbadge.js; ./node_modules/minify/bin/minify build/googlefontsbadge.js build/googlefontsbadge.js"
+    "build": "mkdir build; browserify index.js -s gfBadge -o build/googlefontsbadge.js; ./node_modules/.bin/minify build/googlefontsbadge.js build/googlefontsbadge.js"
   }
 }

